index.html

```diff
@@ -85,14 +85,26 @@ <h4 class="variableTitle">Variables</h4>
             <div class="vizCol">
                 <h4 class="graphsTitle">Graphs</h4>
                 <div class="dataViz">
-                <canvas id="firstGraph"></canvas>
-                <canvas id="secondGraph"></canvas>
+                    <div class="canvas-outer-bar">
+                        <div class="canvas-inner">
+                            <canvas id="firstGraph"></canvas>
+                        </div>
+                    </div>
+                    <div class="canvas-outer-radar">
+                        <div class="canvas-inner">
+                            <canvas id="secondGraph"></canvas>
+                        </div>
+                    </div>
+
+                
                 <canvas id="thirdGraph"></canvas>
                 <script> 
                     let firstGraph = document.getElementById("firstGraph").getContext('2d');
                     let secondGraph = document.getElementById("secondGraph").getContext('2d');
                     let thirdGraph = document.getElementById("thirdGraph").getContext('2d');
 
+                    document.getElementById("firstGraph").height = 400;
```

Comment:
I was playing around and for some reason without this the graph renders very small and the labels are misaligned. You can play around with the number and might be able to find a better value. Also, maybe try applying it to the other two graphs just to see what happens.